src/cdk/activation.ts

```diff
@@ -68,4 +74,15 @@ async function registerCdkCommands(context: vscode.ExtensionContext, explorer: A
             }
         })
     )
+
+    const visualizationManager = new AslVisualizationCDKManager(context)
+    context.subscriptions.push(
+        vscode.commands.registerCommand('aws.cdk.renderStateMachineGraph', async (node: ConstructNode) => {
```

Comment:
Combined with feedback from https://github.com/aws/aws-toolkit-vscode/pull/2005/files#diff-b35a5bc412d7882e5919a26e30a3ed5869615483c52ddf63ec0bf80d1f911c2bR23 : if you just pass `node` whether or not its defined, you can drop the if statement and the boolean at the end. Also: change the constructor to have `node` optional.

Reply:
Simplified the logic quite a bit with this.